torchmultimodal/models/mdetr/text_encoder.py

```diff
@@ -69,6 +69,8 @@ def forward(
     ) -> TransformerOutput:
         encoded = embeddings
         mask = torch.squeeze(attention_mask.to(dtype=torch.bool))
+        if mask.dim() == 1:
```

Comment:
why is dim 1 for bsz 1?

Reply:
Because of the unsqueeze operation just above. It will remove all dimensions of size 1, so in this case batch size dim also gets removed, which we don't want.